run-network/ocrd-all-config.yaml

```
@@ -0,0 +1,4 @@
+environment:
+    # this folder contains the workspaces and must be created by the user
+    data_dir: /tmp/mydata
```

I would prefer to use the/a data-folder in the base by default instead of /tmp/mydata as it is currently. Therefore it could be changed here and data/ should also be created in the Makefile in `network-setup`. I often struggle when I realize I have to create /tmp/mydata when testing
```suggestion
    data_dir: $PWD/data
```

---

Yes, that's much better.

---

I'd also create `data`-dir in network-setup. Marked unresolved for now to remember to discuss this in our next meeting

---

In the local deployment, we default to `$PWD:/data` i.e. without subdirectory. So for consistency this would be better (and no need for an extra mkdir)